lib/dialects/abstract/query-generator.js

```diff
@@ -217,41 +217,48 @@ const QueryGenerator = {
     Parameters: table name + list of hashes of attribute-value-pairs.
    @private
   */
-  bulkInsertQuery(tableName, attrValueHashes, options, rawAttributes) {
+  bulkInsertQuery(tableName, fieldValueHashes, options, fieldMappedAttributes) {
```

Comment:
Might seems like API change but internally this was already the case, `attrValueHashes` are actually values mapped by their field names. Same is true for `rawAttributes`. So I have renamed these arguments